processing/src/main/java/io/druid/query/groupby/epinephelinae/SpillingGrouper.java

```diff
@@ -167,71 +172,104 @@ public void close()
     deleteFiles();
   }
 
+  public List<String> getDictionary()
```

Comment:
Could method name be more explicit it's not just a getter? like "loadDictionaryFromDisk"

Reply:
Good point. Renamed to ```mergeAndGetDictionary``` because this method does loading dictionary from disk as well as merging them with the one in memory. It means it can return a dictionary in memory if any data is not spilled.